meta/documents/changelog_en.md

```diff
@@ -37,7 +37,7 @@
 
 ### Added
 
-* [PLENTY-192] Plentymarkets tags are now supported. FINDOLOGIC will automatically filter by the given tag on tag pages.
+* [PLENTY-192] Plentymarkets tags are now supported. Findologic will automatically filter by the given tag on tag pages.
```

Comment:
If you change the changelogs here, they should also be changed in the [Releases on GitHub](https://github.com/findologic/plugin-plentymarkets-ceres-api/releases).

Reply:
Added post-deploy task